src/test/java/net/snowflake/ingest/streaming/internal/datatypes/IcebergDataTypeIT.java

```diff
@@ -0,0 +1,716 @@
+package net.snowflake.ingest.streaming.internal.datatypes;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.OffsetTime;
+import java.time.ZoneOffset;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import net.snowflake.ingest.TestUtils;
+import net.snowflake.ingest.streaming.SnowflakeStreamingIngestChannel;
+import net.snowflake.ingest.utils.ErrorCode;
+import net.snowflake.ingest.utils.SFException;
+import org.apache.commons.lang3.StringUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+@Ignore("This test can be enabled after server side Iceberg EP support is released")
+public class IcebergDataTypeIT extends AbstractDataTypeTest {
```

Comment:
why not split this into multiple classes?

Reply:
Split into multiple test classes as FDN data type IT did.